packages/tools/devtools/devtools-core/src/test/DataVisualizerGraph.spec.ts

```diff
@@ -19,8 +19,8 @@ import {
 	type FluidObjectValueNode,
```

Comment:
Note that *.test.* files have been renamed to *.spec.* to follow Fluid conventions:

*.spec.* -> mocha
*.test.* -> jest

Reply:
You'll need to update the package's eslint config to reflect this change -> it has some rule overrides for test code.